### PR TITLE
Glpn quickfix

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

**Review by tip3x — APPROVED** [review]
LGTM

File: test/models/test_glpn.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def test_glpn():
 
     model = GLPNForDepthEstimation.from_pretrained("vinvino02/glpn-kitti")
     model = model.eval()
-
+    #
```
Comment (doronHarnoy) [inline test/models/test_glpn.py:19]: we should remove it probably no?

File: onnx2kerastl/converter.py [hunk 2]
```diff
@@ -134,10 +137,15 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
...
-                replace_node = node.attribute[1].g.node
+            cond = layers[node.input[0]][0]
+            if not isinstance(cond, bool) and not isinstance(cond, tf.Tensor) and keras.backend.is_keras_tensor(cond):
+                # the condition in If is a KerasTensor and needs to be evlauated.
+                inpt_sample = [tf.ones(inpt.shape) for inpt in keras_inputs]
+                cond = keras.models.Model(keras_inputs, cond)(inpt_sample)
+                if cond:
+                    replace_node = node.attribute[0].g.node
+                else:
+                    replace_node = node.attribute[1].g.node
```
Comment (doronHarnoy) [inline onnx2kerastl/converter.py:148]: Aren't we missing an else block similar to this one? for the case `cond` is bool or tf.Tensor?
```suggestion
                    replace_node = node.attribute[1].g.node
                else:
                    if all(cond):
                        replace_node = node.attribute[0].g.node
                    else:
                        replace_node = node.attribute[1].g.node
                    
```

**Review by doronHarnoy — APPROVED** [review]
LGTM

File: onnx2kerastl/converter.py [hunk 3]
```diff
@@ -134,10 +137,15 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
     embedding_weights_mapping = {}
     for node_index, node in enumerate(onnx_nodes):
         if node.op_type == 'If':
-            if layers[node.input[0]][0]:
-                replace_node = node.attribute[0].g.node
-            else:
-                replace_node = node.attribute[1].g.node
+            cond = layers[node.input[0]][0]
+            if not isinstance(cond, bool) and not isinstance(cond, tf.Tensor) and keras.backend.is_keras_tensor(cond):
```
[the remark this answers is not in the document]
Comment (tomkoren21) [reply onnx2kerastl/converter.py:141]: not really - the is_keras_tensor raises an error if a non-tensor object is used.

File: onnx2kerastl/converter.py (hunk 2 above)
Comment (tomkoren21) [reply onnx2kerastl/converter.py:148]: wrong identation - fixed.

**Pull request merged**